### ISSUE TITLE
histogram Observe() not compiling on i368

### ISSUE THREAD

**Issue by sentriz** [prometheus/client_golang#465]
```
$ env GOARCH=386 go test ./...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8049bbc]

goroutine 5382 [running]:
runtime/internal/atomic.Xadd64(0xad2860c, 0x2, 0x0, 0x0, 0x40080000)
	/usr/lib/go/src/runtime/internal/atomic/asm_386.s:102 +0xc
hello/vendor/github.com/prometheus/client_golang/prometheus.(*histogram).Observe(0xad285b0, 0x0, 0x40080000)
	/home/senan/go/src/hello/vendor/github.com/prometheus/client_golang/prometheus/histogram.go:272 +0x69
...
```

```
$ env GOARCH=amd64 go test ./...
...
pass
```

the calling code looks like `someHist.Observe(float64(someNumber))` where `someHist` is an `Observer`, created with `NewHistogramVec`, which was then `.With`ed

thank you

**Comment by beorn7** [comment]
Ouch. Thanks for reporting. I could reproduce it without problem. Perhaps a bug in the 386 atomic package? Investigating…

In the meantime, please tag your vendoring to commit 7858729281ec582767b20e0d696b6041d995d5e0  on 386 (that's where the new use of the atomic package for histograms was introduced).

**Comment by beorn7** [comment]
Ugh, that's an easy/stupid one, I just missed the 64bit alignment part again. Fix incoming.

**Comment by sentriz** [comment]
@beorn7 great. thank you very much

**Comment by beorn7** [comment]
OK, fixed by #466 . Phew…
